stellarphot/core.py

```diff
@@ -658,36 +672,26 @@ class PhotometryData(BaseEnhancedTable):
         "passband": None,
```

Comment:
I just caught that "convenience" is misspelled "convience" in line 511.

Reply:
Fixed